tests/containers/docker_image.pm

```diff
@@ -55,54 +55,59 @@ sub run {
     }
 
     for my $i (0 .. $#$image_names) {
-        # Load the image
-        assert_script_run("docker pull $image_names->[$i]", 1000);
-        # Running executables works
-        assert_script_run qq{docker container run --entrypoint '/bin/bash' --rm $image_names->[$i] -c 'echo "I work" | grep "I work"'};
-        # It is the right version
-        if (is_sle) {
-            $osversion = get_required_var("VERSION") =~ s/-SP/ SP/r;    # 15 -> 15, 15-SP1 -> 15 SP1
-            validate_script_output("docker container run --entrypoint '/bin/bash' --rm $image_names->[$i] -c 'cat /etc/os-release'", sub { /PRETTY_NAME="SUSE Linux Enterprise Server $osversion"/ });
+        # Ignore internal images on public cloud
+        if (is_public_cloud && $image_names->[$i] =~ /registry.suse.[de|com]/) {
```

Comment:
`registry.suse.de` is not available in public network without VPN, Ok to skip it, but why not `registry.suse.com`?

Reply:
This got resolved by extending `get_suse_container_urls` which made this check obsolete